test/object_store/test_basic.py

```diff
@@ -220,3 +224,56 @@ def list_bucket(s3_server):
         for o in objects:
             for ent in sstable_entries:
                 assert not o.startswith(str(ent[1])), f'Sstable object not cleaned, found {o}'
+
```

Comment:
could you please add two empty lines instead of one? see https://peps.python.org/pep-0008/#blank-lines